docs/plugin-syntax-decorators.md

```diff
@@ -40,8 +40,26 @@ require("@babel/core").transformSync("code", {
 
 ## Options
 
+<details>
```

Comment:
Since the usage of `syntax-decorators` is rare, we can just leave a "see also" redirect link so we don't have to maintain the history, example, etc.